Congrats.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { StyleSheet, Text, View, Button, Image, Dimensions, TouchableOpacity } from 'react-native';
+import { Font } from 'expo';
+import PongButton from './PongButton';
+
+
+
+export class Congrats extends React.Component {
+  state = { fontLoaded: false };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble-shadow': require('./assets/fonts/Double_Bubble_shadow.otf'),
+      'source-sans-pro': require('./assets/fonts/SourceSansPro-Bold.ttf'),
+    });
+    this.setState({ fontLoaded: true });
+  }
+  render() {
+    return (
+      <View style={{ flexDirection: 'column' }}>
+        <View style={{ height: '58%', alignItems: 'center' }}>
+          {
+            this.state.fontLoaded ? (
```

Comment:
**Code Reduction**

This `if` statement is repeating a lot of code. What you can do instead to save space is do the following in the `Text` components:

`<Text style={ this.state.fontLoaded ? styles.congratsText : styles.congratsTextBackup }>...</Text>`

This will definitely reduce a lot of extra code